src/ontology/mondo-edit.obo

```diff
@@ -151202,35 +151208,34 @@ property_value: seeAlso https://rarediseases.info.nih.gov/diseases/10801/medulla
 
 [Term]
 id: MONDO:0008265
-name: isolated polycystic liver disease
+name: polycystic liver disease 1
```

Comment:
Note the name no longer matches the def

I'll merge, but make a new ticket

Reply:
I revised the def.